### PR TITLE
qm-importdisk: add page

### PR DESCRIPTION
**Pull request by siikanen**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Related  to issue _Let's document: qm_ https://github.com/tldr-pages/tldr/issues/8537

### CODE REVIEW

File: pages/linux/qm-importdisk.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# qm importdisk
+
+> Import disk image to a VM as unused disk.
+> The image format has to be supported by qemu-img(1) such as raw, qcow2, qed, vdi, vmdk, vhd.
```
Comment (kbdharun) [inline pages/linux/qm-importdisk.md:4]:
```suggestion
> The image format has to be supported by `qemu-img` such as raw, qcow2, qed, vdi, vmdk, vhd.
```

File: pages/linux/qm-importdisk.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Import a VMDK disk image to storage called "local-zfs":
+
+`qm importdisk {{100}} name-of-disk.vmdk local-zfs`
```
Comment (kbdharun) [inline pages/linux/qm-importdisk.md:9]:
```suggestion
`qm importdisk {{vm_id}} {{disk.vmdk}} {{local-zfs}}`
```

File: pages/linux/qm-importdisk.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+- Import a VMDK disk image to storage called "local-lvm" as qcow2 format:
+
+`qm importdisk {{100}} name-of-disk.vmdk local-lvm --format qcow2`
```
Comment (kbdharun) [inline pages/linux/qm-importdisk.md:13]:
```suggestion
`qm importdisk {{vm_id}} {{disk.vmdk}} {{local-lvm}} --format {{qcow2}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/qm-importdisk.md (hunk 2 above)
Comment (navarroaxel) [reply pages/linux/qm-importdisk.md:9]: This command requires the path to the vmdk file, right? Is an storage arbitrary name?
```suggestion
`qm importdisk {{100}} {{path/to/disk.vmdk}} {{storage_name}}`
```

File: pages/linux/qm-importdisk.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# qm importdisk
+
+> Import disk image to a VM as unused disk.
+> The image format has to be supported by qemu-img(1) such as raw, qcow2, qed, vdi, vmdk, vhd.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Import a VMDK disk image to storage called "local-zfs":
```
Comment (kbdharun) [inline pages/linux/qm-importdisk.md:7]:
```suggestion
- Import a VMDK disk image to specific storage:
```

File: pages/linux/qm-importdisk.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+`qm importdisk {{100}} name-of-disk.vmdk local-zfs`
+
+- Import a VMDK disk image to storage called "local-lvm" as qcow2 format:
```
Comment (kbdharun) [inline pages/linux/qm-importdisk.md:11]:
```suggestion
- Import a VMDK disk image to specific storage in a specific format:
```

**Review by kbdharun — APPROVED** [review]

File: pages/linux/qm-importdisk.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# qm importdisk
+
+> Import disk image to a VM as unused disk.
+> The image format has to be supported by `qemu-img` such as raw, qcow2, qed, vdi, vmdk, vhd.
```
Comment (kbdharun) [inline pages/linux/qm-importdisk.md:4]:
```
> The image format has to be supported by `qemu-img` such as raw, qcow2, qed, vdi, vmdk, vhd.
```
If necessary we can rephrase this line.

File: pages/linux/qm-importdisk.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Import a VMDK disk image to specific storage:
+
+`qm importdisk {{100}} {{path/to/disk.vmdk}} {{storage_name}}`
+
+- Import a VMDK disk image to specific storage in a specific format:
+
+`qm importdisk {{vm_id}} {{path/to/disk.vmdk}} {{storage_name}} --format {{qcow2 | raw | vmdk}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/qm-importdisk.md:13]:
```suggestion
`qm importdisk {{vm_id}} {{path/to/disk.vmdk}} {{storage_name}} --format {{qcow2|raw|vmdk}}`
```

Mandatory fix: [remove extra spaces around pipe symbol: `--format` values doesn't support `"qcow2 "`, `" raw "`, and `" vmdk"` values](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

**Pull request closed without merging**